src/ContainsEdititem.js

```diff
@@ -1,14 +1,14 @@
 import { connect } from "react-redux";
 import Edititem from "./Edititem";
-import { editTodo } from "./Actions";
+//import { editTodo } from "./Actions";
 
 const mapStateToProps = state => ({
   todo: state.todo
 });
 const mapDispatchToProps = dispatch => ({
-  editTodo: (text, id) => {
+  /* editTodo: (text, id) => {
     dispatch(editTodo(text, id));
-  }
+  } */
 });
 
```

Comment:
ok, this practice of using exporting is ok, but you should use this mapDisptachToProps in the same file where your main component  EditItem.